bloom-housing/bloom
# 2140/fix paper application submission

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2140)


## Description
On the partner's site, as you add a paper application to a listing, if you only selected one preferred unit when you went to submit this caused an error that locked up the form

**cypress tests will be added in a stand alone pr for a different ticketo this sprint**

## How Can This Be Tested/Reviewed?
Create a new listing (or use an available listing)
on the partner's site navigate to the listing's applications, and add a new application
select 1 preferred unit (and any other info required to submit)
save

this should allow you to save successfully and the result should contain the preferred unit correctly

try again but this time select multiple preferred units

this should allow you to save successfully and the result should contain the preferred units correctly 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
